bin/index.js

```diff
@@ -1,288 +1,220 @@
-        '        await users.delete({\n' +
-        '            userId: user1.userId,\n' +
-        '        });\n' +
-        '        await users.delete({\n' +
-        '            userId: user2.userId,\n' +
-        '        });\n' +
-        '        console.log("users deleted");\n' +
-        '\n' +
-        '        // drop database\n' +
-        '        await this.tigris.dropDatabase("hello_db");\n' +
-        '        console.log("database dropped");\n' +
-        '    }\n' +
-        '}\n';
-    fs.writeFileSync(outputDir + '/src/app.ts', content);
+function initializeCollectionsFile (outputDir) {
+  const content = 'import {User} from "../models/user";\n' +
+    'import {Collection, DB} from "@tigrisdata/core";\n' +
+    'import {SelectorFilterOperator, UpdateFieldsOperator} from "@tigrisdata/core/dist/types";\n' +
+    'import {SearchRequest, SearchResult} from "@tigrisdata/core/dist/search/types";\n' +
+    '\n' +
+    'export class Users {\n' +
+    '  private readonly users: Collection<User>;\n' +
+    '\n' +
+    '  constructor(db: DB) {\n' +
+    '    this.users = db.getCollection<User>("users");\n' +
+    '  }\n' +
+    '\n' +
+    '//TODO: Add CRUD operations here\n' +
+    '}';
+  fs.writeFileSync(outputDir + '/src/collections/users.ts', content)
 }
 
-function initializeConfigFile(outputDir) {
-    const content = 'import {Tigris} from "@tigrisdata/core";\n' +
-        '\n' +
-        'export class Config {\n' +
-        '    public initializeTigrisClient(): Tigris {\n' +
-        '        return new Tigris({\n' +
-        '            serverUrl: "localhost:8081",\n' +
-        '            insecureChannel: true,\n' +
-        '        });\n' +
-        '    }\n' +
-        '}';
-    fs.writeFileSync(outputDir + '/src/lib/config.ts', content);
+function initializeScriptFile (outputDir) {
+  fs.writeFileSync(outputDir + '/src/testScript.ts', '')
 }
 
-function initializeModels(outputDir) {
-    const content = 'import {\n' +
-        '    TigrisCollectionType,\n' +
-        '    TigrisDataTypes,\n' +
-        '    TigrisSchema\n' +
-        '} from "@tigrisdata/core/dist/types";\n' +
-        '\n' +
-        'export interface User extends TigrisCollectionType {\n' +
-        '    userId?: number;\n' +
-        '    name: string;\n' +
-        '    balance: number;\n' +
-        '}\n' +
-        '\n' +
-        'export const userSchema: TigrisSchema<User> = {\n' +
-        '    userId: {\n' +
-        '        type: TigrisDataTypes.INT32,\n' +
-        '        primary_key: {\n' +
-        '            order: 1,\n' +
-        '            autoGenerate: true,\n' +
-        '        },\n' +
-        '    },\n' +
-        '    name: {\n' +
-        '        type: TigrisDataTypes.STRING,\n' +
-        '    },\n' +
-        '    balance: {\n' +
-        '        type: TigrisDataTypes.NUMBER,\n' +
-        '    },\n' +
-        '};\n'
+function initializeModels (outputDir) {
+  const content = 'import {\n' +
```

Comment:
Is it easier to have the content in a file that you then read here.

Reply:
This seems a little complicated for now, will need more investment